ml-algorithms/src/test/java/org/opensearch/ml/engine/MLEngineTest.java

```diff
@@ -54,7 +56,7 @@ public class MLEngineTest {
     @Before
     public void setUp() {
         Encryptor encryptor = new EncryptorImpl("m+dWmfmnNRiNlOdej/QelEkvMTyH//frS2TBeS2BP4w=");
-        mlEngine = new MLEngine(Path.of("/tmp/test" + UUID.randomUUID()), encryptor);
+        MLEngine mlEngine = new MLEngine(Path.of("/tmp/test" + UUID.randomUUID()), encryptor);
```

Comment:
Why do we need to do this? We already defined this outside of setup: `private MLEngine mlEngine;`

Reply:
I hadn't seen the declaration earlier, just fixed the issue. Thank you!